Lua/Plugins/typescript.lua

```diff
@@ -0,0 +1,169 @@
+highlight("else", "reserved")
+highlight("enum", "reserved")
+highlight("export", "reserved")
+highlight("extends", "reserved")
+highlight("false", "reserved")
+highlight("finally", "reserved")
+highlight("for", "reserved")
+highlight("function", "reserved")
+highlight("if", "reserved")
+highlight("import", "reserved")
+highlight("in", "reserved")
+highlight("instanceof", "reserved")
+highlight("new", "reserved")
+highlight("null", "reserved")
+highlight("return", "reserved")
+highlight("super", "reserved")
+highlight("switch", "reserved")
+highlight("this", "reserved")
+highlight("throw", "reserved")
+highlight("true", "reserved")
+highlight("try", "reserved")
+highlight("typeof", "reserved")
+highlight("var", "reserved")
+highlight("void", "reserved")
+highlight("while", "reserved")
+highlight("with", "reserved")
+highlight("yield", "reserved")
+highlight("let", "reserved")
+highlight("interface", "reserved")
+highlight("type", "reserved")
+highlight("async", "reserved")
+highlight("await", "reserved")
+highlight("public", "reserved")
+highlight("private", "reserved")
+highlight("protected", "reserved")
+highlight("readonly", "reserved")
+highlight("static", "reserved")
+
+--- Arithmetic Operators
+highlight("+", "operator")
+highlight("-", "operator")
+highlight("*", "operator")
+highlight("/", "operator")
+highlight("%", "operator")
+highlight("**", "operator")
+
+--- Assignment Operators
+highlight("=", "operator")
+highlight("+=", "operator")
+highlight("-=", "operator")
+highlight("*=", "operator")
+highlight("/=", "operator")
+highlight("%=", "operator")
+highlight("**=", "operator")
+highlight("&=", "operator")
+highlight("|=", "operator")
+highlight("^=", "operator")
+highlight("<<=", "operator")
+highlight(">>=", "operator")
+highlight(">>>=", "operator")
+highlight("&&=", "operator")
+highlight("||=", "operator")
+highlight("??=", "operator")
+
+--- Comparison Operators
+highlight("==", "operator")
+highlight("===", "operator")
+highlight("!=", "operator")
+highlight("!==", "operator")
+highlight(">", "operator")
+highlight("<", "operator")
+highlight(">=", "operator")
+highlight("<=", "operator")
+
+--- Logical Operators
+highlight("&&", "operator")
+highlight("||", "operator")
+highlight("!", "operator")
+highlight("??", "operator")
+
+--- Bitwise Operators
+highlight("&", "operator")
+highlight("|", "operator")
+highlight("^", "operator")
+highlight("~", "operator")
+highlight("<<", "operator")
+highlight(">>", "operator")
+highlight(">>>", "operator")
+
+--- Special Characters
+highlight("{", "special")
+highlight("}", "special")
+highlight("(", "special")
+highlight(")", "special")
+highlight("[", "special")
+highlight("]", "special")
+highlight(".", "special")
+highlight(";", "special")
+highlight(",", "special")
+highlight("<", "special")
+highlight(">", "special")
+highlight(":", "special")
+highlight("?", "special")
+highlight("`", "special")
+highlight("=>", "special")
+
+--- Strings
+highlight_region('"', '"', "string")
```

Comment:
```suggestion
highlight_region('"', '"', "string", true)
```
I suppose this is optional